input/pagecontent/sushi.md

```diff
@@ -119,23 +119,93 @@ Here are some general tips on approaching debugging:
 * If you are getting an error you can't resolve, you can ask for help on the [FHIR Shorthand Chat channel](https://chat.fhir.org/#narrow/stream/215610-shorthand).
 
-### IG Creation
+### IG Development
 
 SUSHI supports publishing implementation guides via the [template-based IG Publisher](https://build.fhir.org/ig/FHIR/ig-guidance/). This section describes the inputs and outputs from this process.
 
+#### Configuration File
+
+The IG Publisher relies on several configuration files, including **ig.ini**, **package-list.json**, **menu.xml**, and an ImplementationGuide resource. Splitting information among multiple files and managing different formats makes IG configuration difficult to manage.
+
+SUSHI offers a single configuration file, in consistent format, with no duplication of information. For ease of use, this file is written using YAML.
+
+Here is a simple example of a config.yaml file:
+
+```yaml
+id: fhir.us.example
+canonical: http://hl7.org/fhir/us/example
+name: ExampleIG
+title: "HL7 FHIR Implementation Guide: Example IG Release 1 - US Realm | STU1"
+description: Example IG exercises many of the fields in a SUSHI configuration.
+status: active
+license: CC0-1.0
+version: 1.0.0
+fhirVersion: 4.0.1
+template: hl7.fhir.template#0.0.5
+copyrightYear: 2019+
+releaseLabel: STU1
+publisher:
+  name: HL7 FHIR Management Group
+  url: http://www.hl7.org/Special/committees/fhirmg
+  email: fmg@lists.HL7.org
+dependencies:
+  hl7.fhir.us.core: 3.1.0
+```
+
+In YAML:
+
+* White space (new lines and indentation) is significant
+* Strings do not have to be quoted unless they contain reserved characters, such as colon (:)
+* Arrays/sequences are created using indentation or `-`
+
+Most properties that can be used in the SUSHI configuration file come directly from the [Implementation Guide resource](https://www.hl7.org/fhir/implementationguide.html#resource). Below are properties that can be used in SUSHI configuration that differ from the resource.
+
+| Property  | Difference from IG resource | Usage   |
+| :---------------------- | :-------------------------- |:---------|
+| meta | none | Used as specified the IG resource |
+| implicitRules | none | Used as specified the IG resource |
+| language | none | Used as specified the IG resource |
+| text | none | Used as specified the IG resource |
+| contained | none | Used as specified the IG resource |
+| extension | none | Used as specified the IG resource |
+| modifierExtension | none | Used as specified the IG resource |
+| experimental | none | Used as specified the IG resource |
+| useContext | none | Used as specified the IG resource |
+| copyright | none | Used as specified the IG resource |
+| packageId | none | Used as specified the IG resource |
+| canonical | N/A | The canonical URL to be used throughout the IG |
```

Comment:
Probably makes sense to move `canonical` to either the very top or the first `N/A` property (above `template`).

Reply:
Would it be above `menu` since? Or actually should `template` get moved down to above `menu` also? So it would be `releaseLabel`, `template`, `canonical`, `menu`?